debian-wallpaper-theme/action-handler.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/bash
+#
+# Custom action handler for changing the wallpaper using the 
+# default wallpapers available in Debian
+# This was tested on Debian 12, after installing Torizon Connector
+#
+
+case "$1" in
+    get-firmware-info)
+        # Perform normal processing for this action.
+        if [ -f ${SECONDARY_FIRMWARE_PATH} ]; then
+                echo '{"status": "ok", "message": "Current background theme: '$(cat ${SECONDARY_FIRMWARE_PATH})'"}'
+                exit 0
+        else
+                echo '{"status": "ok", "message": "Currently there is no previous package"}'
+                exit 0
+        fi
+        exit 64
+        ;;
+    install)
+        # Perform normal processing for this action.
+        NEW_THEME_NAME=$(cat ${SECONDARY_FIRMWARE_PATH})
+        NEW_BACKGROUND_PATH=/usr/share/desktop-base/${NEW_THEME_NAME}/wallpaper/gnome-background.xml
+        if [ -f ${NEW_BACKGROUND_PATH} ]; then
+                sudo -u lava-user DBUS_SESSION_BUS_ADDRESS="unix:path=/run/user/$(id -u lava-user)/bus" gsettings set org.gnome.desktop.background picture-uri 'file://'${NEW_BACKGROUND_PATH}
```

Comment:
Does this user exist on all debian installs or is there a package prerequisite we should mention"